fairscale/nn/pipe/async_schedule.py

```diff
@@ -194,13 +194,7 @@ def run_invocation(
         from .multiprocess_pipeline import create_task
 
         task = create_task(
-            PipelineStyle.AsyncSchedule,
-            self.checkpoint_stop,
-            batch.index,
-            self.group.rank(),
-            batch,
-            partition.module,
-            skip_trackers,
+            self.checkpoint_stop, batch.index, self.group.rank(), batch, partition.module, skip_trackers,
```

Comment:
I think this is needed in the multiprocess_pipeline.py function in the run() function to distinguish the even-loop and non-event-loop cases.

Reply:
Yes. But it is not used inside create_task.